plugin-flex-ts-template-v2/src/utils/serverless/ApiService/index.ts

```diff
@@ -1,9 +1,12 @@
 import * as Flex from '@twilio/flex-ui';
-import { random } from 'lodash';
```

Comment:
I'm not clear on the merits of removing this rather than just using the max vs min entries when using it.

Reply:
This was replaced with `Math.random()`. That provides us with the "full jitter" algorithm from [this article](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/), which is the same algorithm used by the twilio-node client for exponential backoff.

Reply:
As for the merits of full jitter versus a smaller range of jitter, the simulations in the article show that there are actually no benefits when capping the jitter range--the performance is actually better with full jitter.